### PR TITLE
[CPU] MHA will produce NAN when all inputs are negative

### PR DESCRIPTION
**Pull request by luo-cheng2021**
### Details:
 - *MHA will produce NAN when all inputs are negative*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Comment by dmitry-gorokhov** [comment]
@a-sidorova May I ask you to take a look on this issue from Snippets perspective? I am wondering should we fix code-generation pipeline as well.

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp [hunk 1]
```diff
@@ -187,18 +187,23 @@ class MHATest : public testing::WithParamInterface<MHATuple>,
         for (int i = 0; i < funcInputs.size(); ++i) {
             const auto& funcInput = funcInputs[i];
             ov::Tensor tensor;
-            tensor = ov::test::utils::create_and_fill_tensor_normal_distribution(funcInput.get_element_type(), targetInputStaticShapes[i], 1.0f, 0.5f);
+            // TODO: after snippets fixed should remove 2nd condition
```
Comment (a-sidorova) [inline src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp:190]: Could you create the ticket for Snippets and assign to me? It'd be great if the ticket number is in comment. Thank you!

**Review by a-sidorova — APPROVED** [review]

File: src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp (hunk 1 above)
Comment (luo-cheng2021) [reply src/plugins/intel_cpu/tests/functional/subgraph_tests/src/mha.cpp]: Done, ticket: 105339

**Comment by akladiev** [comment]
This PR will be closed in 2 weeks in case of no activity.

**Comment by dmitry-gorokhov** [comment]
@luo-cheng2021 Could you please rebase the PR on latest master?

**Comment by luo-cheng2021** [comment]
> @luo-cheng2021 Could you please rebase the PR on latest master?

Done.

**Review by dmitry-gorokhov — APPROVED** [review]

**Pull request merged**